overlay.go

```diff
@@ -84,6 +84,9 @@ type OverlayConnection interface {
 
 	// DisplayName returns the user-facing overlay name.
 	DisplayName() string
+
+	// DisplayData returns some further data that users may wish to check or monitor
+	DisplayData() map[string]interface{}
```

Comment:
Humble nitpicking: s/`DisplayData`/`Attrs`, as a) `Display` is redundant; b) `Data` is very ambiguous.